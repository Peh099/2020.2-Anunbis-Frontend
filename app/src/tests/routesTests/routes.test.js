import Routes from '../../routes/routes';
import renderer from 'react-test-renderer';

describe('routes test', () =>{
    it('matches the snapshot', () => {
        const tree = renderer.create(<Routes />).toJSON()
        expect(tree).toMatchSnapshot()
    })
    it('matches the snapshot', () => {
        expect(Routes).toMatchSnapshot()
    })
})