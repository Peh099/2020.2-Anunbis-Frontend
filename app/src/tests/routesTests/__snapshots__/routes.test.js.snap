// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`routes test matches the snapshot 1`] = `
<div
  className="sc-jQAxuV ebjiKP"
>
  <div
    className="sc-gGLxEB jopzmU"
  >
    <div
      className="sc-ckTSus iosamV"
    >
      <img
        alt="logo"
        className="sc-fbIWvP cedweP"
        onClick={[Function]}
        src="Name_Logo.png"
      />
    </div>
    <div
      className="sc-GvhzO jqvlLo"
    >
      <div>
        <a
          className="sc-gsWcmt ewTZsD"
          href="#begin"
        >
          Inicio
        </a>
        <a
          className="sc-gsWcmt jEvov"
          href="#howItWorks"
        >
          Como Funciona?
        </a>
        <a
          className="sc-gsWcmt jEvov"
          href="#team"
        >
          Equipe
        </a>
        <a
          className="sc-gsWcmt jEvov"
          href="#contact"
        >
          Contato
        </a>
      </div>
    </div>
    <div
      className="sc-lbVvki guSlFp"
      height="22,5px"
    >
      <button
        className="sc-gKAaRy jXHVdm sc-bXexck daiLzK"
        onClick={[Function]}
      >
        Login
      </button>
    </div>
  </div>
  <main
    className="sc-fuISkM iNIEJi"
  >
    <span
      style={
        Object {
          "height": "100%",
          "width": "100%",
        }
      }
    >
      <div
        className="sc-eGJWMs gKSkYZ"
        display="flex"
        id="begin"
      >
        <div
          className="sc-bsatvv dEeVUX"
        >
          <label>
            Onde a uni√£o faz a for√ßa
          </label>
          <p>
            Alunos trocando experi√™ncias e opini√µes sobre disciplinas e professores para manter a comunidade acad√™mica da Universidade de Brasilia unida e bem informada.
          </p>
          <button
            className="sc-gKAaRy dtRqWS sc-bXexck dHtymf"
            onClick={[Function]}
            width="160px"
          >
            Cadastre-se
          </button>
        </div>
        <img
          alt="Logo"
          className="sc-dFRpbK lgIuap"
          height="80%"
          src="LogoHome.svg"
        />
      </div>
    </span>
    <span
      style={
        Object {
          "height": "100%",
          "width": "100%",
        }
      }
    >
      <div
        className="sc-eGJWMs gKSkYZ"
        display="flex"
        id="howItWorks"
      >
        <div
          className="sc-bsatvv cJvJgc"
        >
          <label>
            Como funciona?
          </label>
          <p>
            Os alunos podem escrever avalia√ß√µes p√∫blicas sobre os professores optando por manter ou n√£o o anonimato. Os demais alunos podem ver as avalia√ß√µes, concordar ou discordar. Todas as notas e coment√°rios s√£o de livre acesso para todos os usu√°rios, incluindo professores.
          </p>
        </div>
        <img
          alt="Puzzle"
          className="sc-dFRpbK fLOjvO"
          height="80%"
          src="Puzzle.svg"
        />
      </div>
    </span>
    <span
      style={
        Object {
          "height": "100%",
          "width": "100%",
        }
      }
    >
      <div
        className="sc-eGJWMs jqoNOx"
        direction="column"
      >
        <div
          className="sc-eGJWMs ekbcJt"
          display="flex"
          height="50%"
          id="team"
        >
          <div
            className="sc-bsatvv cJvJgc"
          >
            <label>
              Equipe
            </label>
            <p>
              O Anunbis √© um projeto criado e desenvolvido por alunos de M√©todos de Desenvolvimento de Software (MDS) da Faculdade do Gama (FGA), da Universidade de Bras√≠lia (UnB).
            </p>
          </div>
          <img
            alt="Team"
            className="sc-dFRpbK pSAXT"
            height="75%"
            src="Teamwork.svg"
          />
        </div>
        <div
          className="sc-eGJWMs igdiVQ"
          display="flex"
          height="100px"
        >
          <div
            className="sc-bsatvv cJvJgc"
          >
            <label>
              Membros
            </label>
          </div>
        </div>
        <div
          className="sc-csTbgd ghDVfN"
        >
          <div
            className="sc-gIvpjk hLgyqR"
          >
            <a
              href="https://github.com/RcleydsonR"
              rel="noreferrer"
              target="_blank"
            >
              <div>
                Rafael Cleydson
              </div>
              <img
                alt="Rafael Cleydson"
                src="Rafael.jpeg"
              />
            </a>
          </div>
          <div
            className="sc-gIvpjk hLgyqR"
          >
            <a
              href="https://github.com/Balbinoo"
              rel="noreferrer"
              target="_blank"
            >
              <div>
                Rodrigo Balbino
              </div>
              <img
                alt="Rodrigo Balbino"
                src="Rodrigo.jpeg"
              />
            </a>
          </div>
          <div
            className="sc-gIvpjk hLgyqR"
          >
            <a
              href="https://github.com/mangabeiras"
              rel="noreferrer"
              target="_blank"
            >
              <div>
                Roberto Santana
              </div>
              <img
                alt="Roberto Santana"
                src="Roberto.jpeg"
              />
            </a>
          </div>
          <div
            className="sc-gIvpjk hLgyqR"
          >
            <a
              href="https://github.com/oEduardoAfonso"
              rel="noreferrer"
              target="_blank"
            >
              <div>
                Eduardo Afonso
              </div>
              <img
                alt="Eduardo Afonso"
                src="Eduardo.jpeg"
              />
            </a>
          </div>
          <div
            className="sc-gIvpjk hLgyqR"
          >
            <a
              href="https://github.com/thiagohdaqw"
              rel="noreferrer"
              target="_blank"
            >
              <div>
                Thiago Paiva
              </div>
              <img
                alt="Thiago Paiva"
                src="Thiago.jpeg"
              />
            </a>
          </div>
          <div
            className="sc-gIvpjk hLgyqR"
          >
            <a
              href="https://github.com/victorhugo21"
              rel="noreferrer"
              target="_blank"
            >
              <div>
                Victor Hugo
              </div>
              <img
                alt="Victor Hugo"
                src="Victor.jpeg"
              />
            </a>
          </div>
        </div>
      </div>
    </span>
    <span
      style={
        Object {
          "height": "100%",
          "width": "100%",
        }
      }
    >
      <div
        className="sc-eGJWMs gKSkYZ"
        display="flex"
        id="contact"
      >
        <div
          className="sc-bsatvv cJvJgc"
        >
          <label>
            Contate-nos
          </label>
          <p>
            Deseja elogiar, sugerir mudan√ßas, ou contribuir? Entre em contato conosco atrav√©s do email ou acesse nosso reposit√≥rio no github.
          </p>
          <a
            href="mailto:anunbis.team@gmail.com"
          >
            anunbis.team@gmail.com
          </a>
          <a
            href="https://github.com/fga-eps-mds/2020.2-Anunbis"
            rel="noreferrer"
            target="_blank"
          >
            Reposit√≥rio Back-end
          </a>
          <a
            href="https://github.com/fga-eps-mds/2020.2-Anunbis-FrontEnd"
            rel="noreferrer"
            target="_blank"
          >
            Reposit√≥rio Front-end
          </a>
        </div>
        <img
          alt="Img"
          className="sc-dFRpbK LyVoj"
          height="50%"
          src="Communication.svg"
        />
      </div>
    </span>
    <main
      className="sc-fcmMJX ljRJGn"
    >
      <p>
        üÑØ Anunbis 2021
      </p>
      <div>
        Icons made by
        <a
          href="https://www.freepik.com"
          title="Freepik"
        >
          Freepik
        </a>
         
        from
         
        <a
          href="https://www.flaticon.com/"
          title="Flaticon"
        >
          www.flaticon.com
        </a>
      </div>
    </main>
  </main>
</div>
`;

exports[`routes test matches the snapshot 2`] = `[Function]`;
